frontend/src/components/header/Header.tsx

```diff
@@ -31,10 +31,10 @@ export const Header = () => {
     return (
         <div className="header">
             <div className="header-logo">
-                <Button text={"Bucket Flowers"} onClick={() => navigate("/")} />
+                <Button text={"Bucket flowers"} onClick={() => navigate("/")} className='btn-home' />
```

Comment:
don't like the idea of using this class here

Reply:
why? both buttons have almost equal styles, i just modify it to look more like an icon, not a button

Reply:
Its about the usage of this class.. The `btn-home` class is not ours, so in the future there might be issues with that. For example after package update.